mrmustard/lab_dev/circuit_components.py

```diff
@@ -589,10 +592,6 @@ def __init__(self, component: CircuitComponent) -> None:
         self.__dict__ = component.__dict__.copy()
         self._component = component._light_copy()
 
-    def __getattr__(self, name):
```

Comment:
@ziofil Can you confirm this can be removed? It doesn't seem to break any tests but just want to make sure

Reply:
Yes, it should be fine to remove because it's not doing what it should ^^'
The need was to avoid creating copies of large arrays (which may exist when the representation is `Fock`), so we had created a view that lazily conjugates the representation, and the `__getattr__` was supposed to make it behave like a regular CircuitComponent. But we didn't implement it correctly: it shouldn't look for the method in `_component` (otherwise you get the bug you discovered that `.inverse()` keeps getting called on the original). It should look for the method in `super()` and super should be the same class where `.dual` or `.adjoint` was called. What do you think?

Reply:
Discussed offline -  this is to be investigated and done in a follow up PR